packages/camltc/camltc.0.9.8/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "romain.slootmaekers@openvstorage.com"
+authors: ["Jan Doms" "Joost Damad" "Romain Slootmaekers" "Nicolas Trangez"]
+homepage: "http://github.com/toolslive/camltc"
+bug-reports: "http://github.com/toolslive/camltc/issues"
+dev-repo: "https://github.com/toolslive/camltc.git"
+license: "Apache-2.0"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+build-test: [
+  ["dune" "build" "@runtest" "-j" jobs]
```

Comment:
I think the idiomatic way is `["dune" "runtest" "-p" name "-j" jobs]`, they are nearly equivalent but specifying the name will restrict the compiled units and tests to the relevant ones. i suggest the idiomatic form only because using that uncovered some transitive dependencies issues in packages that I was maintaining in the past

Reply:
I've updated the rule for build-test to follow your suggestion. 
thanks for the comments.